### DISCUSSION TITLE
Struggling with LESS

### DISCUSSION THREAD

**Discussion by glow12121** [paulradzkov/bettertext.css#8] [Q&A] [answered]
A really ignorant question:

Because I could not get LESS working in Jekyll, I tried to make CSS calculations online using [the link you kindly provided](https://codepen.io/paulradzkov/pen/Mygdrb).

However, as I change the default font to `@fz: 20px;`, the text in the preview section gets smaller, not bigger. Could I be doing something wrong?

![Screenshot From 2024-11-08 20-23-13](https://github.com/user-attachments/assets/9684e910-608e-4c13-bd42-35bf613579e3)

![Screenshot From 2024-11-08 20-23-02](https://github.com/user-attachments/assets/09213a57-5f6c-4359-8621-ef6f05dad174)

**Comment by paulradzkov** [comment]
That's my bad! 
I have fixed a silly mistake at line 33 on [codepen](https://codepen.io/paulradzkov/pen/Mygdrb) playground.
I'm going to test a little bit more and release a patch.

Thank you for raising the problem!

**Reply by glow12121** [reply]
> I'm going to test a little bit more and release a patch.

Great, many thanks.

**Comment by paulradzkov** [comment, marked as answer]
Updated to v2.0.5
